packages/graphql/src/translate/cypher-builder/examples/create.test.ts

```diff
@@ -21,31 +21,27 @@ import * as CypherBuilder from "../CypherBuilder";
 
 describe("CypherBuilder Create Examples", () => {
     test("Create a movie", () => {
-        const titleParam = new CypherBuilder.Param("The Matrix");
```

Comment:
I think that the test was better before with both the `title` and the `released` parameter!